V2/tests/integration/daos/customers_dao_test.py

```diff
@@ -1,4 +1,6 @@
 import pytest
+import typing
+from faker import Faker
```

Comment:
imports should be

- System imports (os, typing, etc)
- One space
- Library imports (flask, azure, faker,  etc)
- One space
- App imports (form ._time_entries import something)